KBH/12Day/실패율.js

```diff
@@ -0,0 +1,35 @@
+function solution(N, stages) {
+    const answer = [];
+    const copyedStages = [...stages];
+    //실패한 스테이지 개수 확인
+    const failedStage = copyedStages.reduce((acc, cur) => { acc[cur] = (acc[cur] || 0 ) + 1; 
+        return acc;
+    }, {});
+    
+    //다음 스테이지에 올라간 유저수
+    let stageLength = stages.length;
+    //실패율 변수 선언
+    let failedRate = [];
```

Comment:
```suggestion
    const failedRate = [];
```
해당 참조형 변수는 `const`로 선언해주는것이 좋아보입니다!